### PR TITLE
iam: update some descriptions and links

### PR DESCRIPTION
**Pull request by unknwon**

### CODE REVIEW

File: content/departments/engineering/dev/admin-exp/iam/index.md [hunk 1]
```diff
@@ -58,18 +58,15 @@ We’re hiring! [Check out our open roles](https://boards.greenhouse.io/sourcegr
 
 ## Partner Teams
 
-The IAM team works alongside several other teams within the Cloud organization at Sourcegraph. You can find more information about their teams and goals on the respective pages:
+The IAM team works alongside several other teams at Sourcegraph. You can find more information about their teams and goals on the respective pages:
 
-- [Cloud DevOps](../devops)
-- [Integrations](../integrations)
-- [Growth](../growth-team)
```
Comment (kopancek) [inline content/departments/engineering/dev/admin-exp/iam/index.md:65]: Should we mention repo management here?

File: content/departments/engineering/dev/admin-exp/iam/index.md [hunk 2]
```diff
@@ -58,18 +58,15 @@ We’re hiring! [Check out our open roles](https://boards.greenhouse.io/sourcegr
...
 - [Security](../security) - Team ambassador: [Feroz Salam](../../../../../team/index.md#feroz-salam)
 
 ## News and updates
 
 ### Weekly team updates
 
-Each Friday, we send weekly team updates to the rest of the Cloud org to keep the Cloud senior leadership and our partner teams in the loop about IAM Team progress and our ups and downs. You can find all the weekly updates in this [Google Docs](https://docs.google.com/document/d/1a_36dJ-ZR4LU_3bDaIWAZldZJ6O3zaSi8NhGfT0fsdw/edit#heading=h.yggics8n0cyt).
+Each Friday, we send weekly team updates to the rest of the Admin Experience org to keep the senior leadership and our partner teams in the loop about IAM team progress and our ups and downs. You can find all the weekly updates in this [Google Docs](https://docs.google.com/document/d/1a_36dJ-ZR4LU_3bDaIWAZldZJ6O3zaSi8NhGfT0fsdw/edit#heading=h.yggics8n0cyt).
```
Comment (kopancek) [inline content/departments/engineering/dev/admin-exp/iam/index.md]: Is this still true? Effectively, there are no orgs anymore.

**Review by kopancek — APPROVED** [review]

File: content/departments/engineering/dev/admin-exp/iam/index.md (hunk 1 above)
Comment (unknwon) [reply content/departments/engineering/dev/admin-exp/iam/index.md:65]: Good catch, yes!

File: content/departments/engineering/dev/admin-exp/iam/index.md (hunk 2 above)
Comment (unknwon) [reply content/departments/engineering/dev/admin-exp/iam/index.md]: Hmm, yeah, I'll remove it!

**Pull request merged**